src/test/kotlin/chapter8/solutions/ex9/listing.kt

```diff
@@ -24,7 +24,7 @@ data class Falsified(
 
 //tag::init[]
 data class Prop(val run: (TestCases, RNG) -> Result) {
-    fun and(other: Prop) = Prop { n, rng ->
+    fun and(other: Prop): Prop = Prop { n, rng ->
```

Comment:
For me, the type is already explicit because `= Prop {` clearly states that what will be returned is a `Prop`.
If you feel strongly about this we can go with it though.

Reply:
Not so much...! The explicit type can be omitted.

Reply:
I removed the explicit types I added.